crates/driver/src/main.rs

```diff
@@ -87,6 +87,9 @@ pub struct Opt {
     #[clap(long)]
     cache_path: PathBuf,
 
+    #[clap(long)]
+    no_aggregate_source: bool,
```

Comment:
can we add a comment as to what this means? Is it minimize the number of targets or error if you cannot make a DAG with this file in a single node?

Reply:
Addressed in aa1a4a68b019b67a2cd9086ac678cd4408877ff6